TheFourFifths/consus
# [TFF-225] Equipment Unsaving

# >>> REVIEW AFTER TFF-220 <<<

### Summary

We can now retrieve saved equipment.

### Checklist


### How to Review

1. Run both the server and client on `item-save-TFF-225`.
2. Create a model that can be checked out.
3. Check out a model and an item.
4. Save them both.
5. Retrieve them both (by scanning the address into the cart input and/or by clicking "Retrieve").

### Links

- [TFF-225](https://msoese.atlassian.net/browse/TFF-225)
- TheFourFifths/consus-client/pull/120
